### ISSUE TITLE
npx create-shuttle-app results in `sh: 1: create-shuttle-app: not found`

### ISSUE THREAD

**Issue by hobo1618** [shuttle-hq/create-shuttle-app#38]
Hello hello!
I've just tried running `npx create-shuttle-app` but I get this error: `sh: 1: create-shuttle-app: not found`
Any idea what's causing this?
OS: I just upgraded to Ubuntu 22.04.2 LTS
npm version 9.7.2
Any help would be much appreciated!
Thanks in advance

**Comment by oddgrd** [comment]
Something went wrong with the latest release, I'll look into it, thanks!

**Comment by oddgrd** [comment]
It should work now on `npx create-shuttle-app@0.1.21`

**Comment by hobo1618** [comment]
thanks for looking into this. I was able to initialize a project but unable to run it locally or deploy it. I'll post the error as a separate issue though.

**Comment by ZimboPro** [comment]
I had the same issue, only once I downgrade to 0.1.26 did I succeed

**Comment by oddgrd** [comment]
> I had the same issue, only once I downgrade to 0.1.26 did I succeed

Hey, sorry about that, there was an error in the last release. I published 0.1.29 now in which the error is resolved.
